monai/data/dataset.py

```diff
@@ -376,9 +389,12 @@ def __init__(
             label_transform (Callable, optional): transform to apply to label arrays
 
         """
-
         super().__init__(
-            [Dataset(img_files, img_transform), Dataset(seg_files, seg_transform), Dataset(labels, label_transform)]
+            [
```

Comment:
This looks like what black does automatically with overlong lines of this type. I would put the list to be iterated over in a separate variable so that this line fits within the limits rather than have this multi-line expansion, I feel readability suffers when breaking expressions over so many lines.

Reply:
I was going to comment to use `np.iinfo(np.int32).max`, in the `randomize` method just below here, instead of the literal `2147483647`. This makes it more explicit what that's going on though more complicated.

Reply:
Hi @ericspod ,

Thanks for your review.
I updated according to your comments.
Could you please help review it again?
Thanks.